pkg/chain/ethereum/tbtc.go

```diff
@@ -3,10 +3,14 @@ package ethereum
 import (
 	"fmt"
 	"math/big"
+	"sort"
+
+	eth "github.com/keep-network/keep-ecdsa/pkg/chain"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/keep-network/keep-common/pkg/subscription"
 	"github.com/keep-network/tbtc/pkg/chain/ethereum/gen/contract"
+	"github.com/keep-network/tbtc/pkg/chain/ethereum/gen/filterer"
```

Comment:
"filerer" in this context sounds weird IMO. It suggests we are filtering something from tBTC while we only do an event lookup. Looking at https://github.com/keep-network/tbtc/pull/740/, I'd probably go with "event log"

Reply:
The `filterer` naming is borrowed from `go-ethereum`. I wanted to be consistent with the naming of contract backends generated by `abigen`. But agree, `event log` sounds much better.